### PR TITLE
Add ris_proposals listing entry

### PR DESCRIPTION
**Pull request by blu14x**
Adds the listing entry `ris_proposals` so we can list the RisProposals of a Dossier.

This add some changes to the testing fixtures.
Therefore an update to the UI fixtures: https://github.com/4teamwork/gever-ui/pull/2714

For [TI-152]

## Checklist

_Everything has to be done/checked. Checked but not present means the author deemed it unnecessary._

- [x] Changelog entry
- [x] Link to issue (Jira or GitHub) and backlink in issue (Jira)

_Only applicable should be left and checked._

- API change:
  - [x] Documentation is updated
  - [x] API Changelog entry (see [guide](https://4teamwork.atlassian.net/wiki/spaces/4TEAM/pages/451248812/API+Changelog+Guidelines))

[TI-152]: https://4teamwork.atlassian.net/browse/TI-152?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: opengever/testing/fixtures.py [hunk 1]
```diff
@@ -1492,6 +1493,14 @@ def create_tasks(self):
         ))
         self.create_task_subscriptions(inbox_task)
 
+    @staticuid()
```
Comment (phgross) [inline opengever/testing/fixtures.py:1496]: When adding new content to the fixture, there needs to be adjustments also in the gever-ui please create a PR and link them in the PR description.

(see the globalTestFixture e2e tests, a run should add the changes)

File: docs/public/dev-manual/api/listings.rst [hunk 2]
```diff
@@ -59,6 +59,7 @@ Aktuell werden folgende Auflistungen unterstützt:
 - ``proposals``: Anträge
 - ``contacts``: Lokale Kontakte
 - ``repository_folders``: Ordnungspositionen
+- ``ris_proposals``: RIS-Anträge
```
Comment (phgross) [inline docs/public/dev-manual/api/listings.rst:62]: Update in the listing below (line 146) is missing.

File: opengever/api/tests/test_listing.py [hunk 3]
```diff
@@ -998,6 +998,25 @@ def test_tasks_listing(self, browser):
...
+    def test_ris_proposals_listing(self, browser):
+        self.login(self.regular_user, browser=browser)
+        query_string = '&'.join((
+            'name=ris_proposals',
+            'columns=UID',
+            'columns=title',
+        ))
+        view = '?'.join(('@listing', query_string))
+        browser.open(self.dossier, view=view,
+                     headers={'Accept': 'application/json', 'Accept-Language': 'de-ch'})
```
[the remark this answers is not in the document]
Comment (blu14x) [reply opengever/api/tests/test_listing.py]: ah yes 👌

File: docs/public/dev-manual/api/listings.rst (hunk 2 above)
Comment (blu14x) [reply docs/public/dev-manual/api/listings.rst:62]: done 👌

File: opengever/testing/fixtures.py (hunk 1 above)
Comment (blu14x) [reply opengever/testing/fixtures.py:1496]: i see. did that 👍

**Pull request merged**